scripts/teamd_increase_retry_count.py

```diff
@@ -0,0 +1,320 @@
+            if key == "local_addr":
+                if value not in portChannelData:
+                    break
+                localAddr = value.lower()
+            elif key == "admin_status":
+                isAdminUp = value == "up"
+        if not localAddr:
+            continue
+        portChannelData[localAddr]["adminUp"] = isAdminUp
+
+    return set([portChannelData[x]["portChannel"] for x in portChannelData.keys() if portChannelData[x]["adminUp"]])
+
+def getPortChannelConfig(portChannelName):
+    (processStdout, _) = getCmdOutput(["teamdctl", portChannelName, "state", "dump"])
+    return json.loads(processStdout)
+
+def getLldpNeighbors():
+    (processStdout, _) = getCmdOutput(["lldpctl", "-f", "json"])
+    return json.loads(processStdout)
+
+def craftLacpPacket(portChannelConfig, portName, isResetPacket=False, newVersion=True):
+    portConfig = portChannelConfig["ports"][portName]
+    actorConfig = portConfig["runner"]["actor_lacpdu_info"]
+    partnerConfig = portConfig["runner"]["partner_lacpdu_info"]
+    l2 = Ether(dst=SLOW_PROTOCOL_MAC_ADDRESS, src=portConfig["ifinfo"]["dev_addr"], type=LACP_ETHERTYPE)
+    l3 = scapy.contrib.lacp.SlowProtocol(subtype=0x01) 
+    l4 = LACPRetryCount()
+    if newVersion:
+        l4.version = 0xf1
+    else:
+        l4.version = 0x1
+    l4.actor_system_priority = actorConfig["system_priority"]
+    l4.actor_system = actorConfig["system"]
+    l4.actor_key = actorConfig["key"]
+    l4.actor_port_priority = actorConfig["port_priority"]
+    l4.actor_port_number = actorConfig["port"]
+    l4.actor_state = actorConfig["state"]
+    l4.partner_system_priority = partnerConfig["system_priority"]
+    l4.partner_system = partnerConfig["system"]
+    l4.partner_key = partnerConfig["key"]
+    l4.partner_port_priority = partnerConfig["port_priority"]
+    l4.partner_port_number = partnerConfig["port"]
+    l4.partner_state = partnerConfig["state"]
+    if newVersion:
+        l4.actor_retry_count = EXTENDED_RETRY_COUNT if not isResetPacket else DEFAULT_RETRY_COUNT
+        l4.partner_retry_count = DEFAULT_RETRY_COUNT
+    packet = l2 / l3 / l4
+    return packet
+
+def sendLacpPackets(packets, revertPackets):
+    while not revertTeamdRetryCountChanges:
+        for port, packet in packets:
+            sendp(packet, iface=port)
+        time.sleep(15)
+    if revertTeamdRetryCountChanges:
+        for port, packet in revertPackets:
+            sendp(packet, iface=port)
+
+def abortTeamdChanges(signum, frame):
+    log.log_info("Got signal {}, reverting teamd retry count change".format(signum))
+    revertTeamdRetryCountChanges = True
+
+def getCmdOutput(cmd):
+    proc = subprocess.Popen(cmd, stdout=subprocess.PIPE)
+    return proc.communicate()[0], proc.returncode
+
+def main(probeOnly=False):
+    if os.geteuid() != 0:
+        log.log_error("Root privileges required for this operation", also_print_to_console=True)
+        sys.exit(1)
+        return False
```

Comment:
This return will never be reached  after sys.exit

Reply:
Removed the extra returns.